### PR TITLE
oops

### PR DESCRIPTION
**Pull request by oddbookworm**
hopefully temp fix for #74

### CODE REVIEW

File: intro/classes/index.html [hunk 1]
```diff
@@ -6,6 +6,11 @@
 next_page: [/intro/audio, "Audio in Pygame"]
 prev_page: [/intro/images, "Images in Pygame"]
 ---
+<head>
+    <meta charset="utf-8">
+    <title>{{ page.title }}</title>
+    <link rel="stylesheet" href="/assets/css/main.css">
+</head>
 <body>
     <h2>Table of Contents</h2>
```
Comment (Matiiss) [inline intro/classes/index.html:15]: Missing title header in body

**Review by Matiiss — APPROVED** [review]
LGTM except for the title but since this is temporary, not sure how important that is.

**Review by Matiiss — APPROVED** [review]
LGTM

**Review by SSS-Says-Snek — APPROVED** [review]
LGTM too

**Pull request merged**